File: online-judge-solutions/Codeforces/1728A.py
#https://codeforces.com/problemset/problem/1728/A
#Lang		:	Python 3.8
#Time		:	30 ms
#Memory         :	0 KB
for _ in range(int(input())):
    n=int(input())
    a=list(map(int,input().split()))
    print(a.index(max(a))+1)

File: online-judge-solutions/Codeforces/1728A.cpp
//https://codeforces.com/problemset/problem/1728/B
//Lang		:	GNU C++14
//Time		:	0 ms
//Memory	:	0 KB
#include<iostream>
#include<algorithm>
using namespace std;
#define fastInp cin.tie(0); cout.tie(0); ios_base::sync_with_stdio(0);
int main(){
	fastInp;
	int t,n;
	cin>>t;
	while(t--){
		cin>>n;
		int a[n];
		for(int i=0;i<n;i++) cin>>a[i];
		cout<<max_element(a,a+n)-a+1<<endl;
	}
	return 0;
}
